src/pages/landing/ContentSection/QuotesCarousel.tsx

```diff
@@ -4,8 +4,11 @@ import { styled } from 'styled-components';
 import { FONT_MEDIUM, MOBILE_MAX_WIDTH } from '~/components';
 import QUOTES from '~/data/quotes.json';
 
+const quotes = Array(9).fill(QUOTES).flat();
+const initialIndex = QUOTES.length * Math.floor(9 / 2) - 1;
```

Comment:
the opening quote must be from Optimism

Reply:
[a990e36](https://github.com/defi-wonderland/web/pull/122/commits/a990e36a08fcec0cf60402a8337f68122fe59854)
Given order by Lumi/Mati